src/renderer/opengl/CMakeLists.txt

```
@@ -108,6 +108,8 @@ target_sources(${NOX_OPENGL_LIB_NAME} PRIVATE ${NOX_OPENGL_SOURCES})
 
 if(NOX_PLATFORM_WINDOWS)
     add_subdirectory(windows)
+elseif(NOX_PLATFORM_LINUX)
+    add_subdirectory(linux)
 endif()
```

in order to be consistent with what it is in `src/window/CMakeLists.txt`

```suggestion
if(NOX_PLATFORM_WINDOWS)
    add_subdirectory(windows)
endif()

if(NOX_PLATFORM_LINUX)
    add_subdirectory(linux)
endif()
```